Integrations/BeyondTrust_Password_Safe/BeyondTrust_Password_Safe.py

```diff
@@ -0,0 +1,316 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import json
+import requests
+from requests import Request, Session
+
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+USERNAME = demisto.params().get('credentials').get('identifier')
+PASSWORD = demisto.params().get('credentials').get('password')
+API_KEY = demisto.params().get('key')
+# Remove trailing slash to prevent wrong URL path to service
+SERVER = demisto.params()['url'][:-1] \
+    if (demisto.params()['url'] and demisto.params()['url'].endswith('/')) else demisto.params()['url']
+# Should we use SSL
+USE_SSL = not demisto.params().get('insecure', False)
+# Service base URL
+BASE_URL = SERVER + '/BeyondTrust/api/public/v3'
+# Headers to be sent in requests
+HEADERS = {
+    'Content-Type': 'application/json',
+    'Accept': 'application/json',
+    'Authorization': 'PS-Auth key=' + API_KEY + '; ' + 'runas=' + USERNAME + '; ' + 'pwd=[' + PASSWORD + '];'
+}
+
+SESSION = requests.session()
```

Comment:
why are we using a session?

Reply:
We need to use Session According to the API guide